prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -94,12 +94,21 @@ class FHIRRouter(
         "%messageId.exists()",
     )
 
+    /**
+     * Default Rules for quality filter on ELR_ELIMS topic:
+     *   no rules; completely open
+     */
+    private val elrElimsQualityFilterDefault: ReportStreamFilter = listOf(
+        "true",
```

Comment:
I can see this getting to be quite juicy, eventually. But good for the pilot for now